### ISSUE TITLE
When visiting the /conceptschemes route via a browser, a 500 HTTP error is displayed.

### ISSUE THREAD

**Issue by goessebr** [OnroerendErfgoed/pyramid_skosprovider#107]
When visiting the /conceptschemes route (e.g., https://inventaris.onroerenderfgoed.be/conceptschemes) via a browser, a 500 HTTP error is displayed.

However, the route is accessible when an Accept header with the value "application/json" is provided. I would expect a 404 error when requesting a view that does not exist.

**Comment by Wim-De-Clercq** [comment]
>  I would expect a 404 error when requesting a view, f.e. text/html, that does not exist.

That is true, however most browsers sent merely a preference for html. And will in the end also accept `*/*`. So I will implement the 404, but when testing this issue in a browser you will get the JSON response most likely. (which is also a bug, I dont know why it's not returning the JSON by default)

**Comment by Wim-De-Clercq** [comment]
In the case of accept headers which have multiple potential views as a result. [Pyramid chooses one randomly](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/viewconfig.html#accept-header-content-negotiation).

> In these cases the preferred view is not clearly defined (see [RFC 7231#section-5.3.2](https://datatracker.ietf.org/doc/html/rfc7231.html#section-5.3.2)) and Pyramid will select one randomly.

The "randomly" in this case appears to always be the json-ld view.
And the [view-config](https://github.com/OnroerendErfgoed/pyramid_skosprovider/blob/develop/pyramid_skosprovider/views.py#L130) is simply missing a renderer. Which also indicates the json-ld response of this view has never been tested.
